client/src/modules/inventory/list/modals/search.modal.js

```diff
@@ -68,17 +66,17 @@ function InventorySearchModalController(ModalInstance, Notify, filters, Inventor
   };
 
   // returns the parameters to the parent controller
-  function submit(form) {
+  function submit() {
     // push all searchQuery values into the changes array to be applied
-    angular.forEach(vm.searchQueries, function (value, key) {
-      if (angular.isDefined(value)) {
+    angular.forEach(vm.searchQueries, (val, _key) => {
+      if (angular.isDefined(val)) {
         // default to the original value if no display value is defined
-        var displayValue = displayValues[key] || lastDisplayValues[key] || value;
-        changes.post({ key: key, value: value, displayValue: displayValue });
+        const displayVal = displayValues[_key] || val;
+        changes.post({ key : _key, value : val, displayValue : displayVal });
```

Comment:
ESLint will throw an error here because newer JavasScript allows you to combine the key and the value if they are named the same.  Like this:
```js
// before:
var key = '1';
var o =  { key : key };
// o is { 'key' : '1' }

// after
var key = 2;
var o = { key };
// o is now { 'key' : 2 }
```

Reply:
I wonder if I should update my ESlint library in this  case